src/services/dataServices.js

```diff
@@ -0,0 +1,9 @@
+import getCitiesData from './getCitiesData';
+import getData from './getData';
+
+const dataServices = {
```

Comment:
I'd say it is the one service containing two methods. getCitiesData is more like a method for the service